sdk/cosmos/azure-cosmos-kafka-connect/src/docker/Dockerfile.sinkperf

```diff
@@ -0,0 +1,6 @@
+# Use the Kafka client image as base
+FROM confluentinc/cp-kafka:6.0.1
```

Comment:
dito

Reply:
Same as above, tracking item: https://github.com/Azure/azure-sdk-for-java/issues/38770